CurranSeam/Home-Surveillance-System
# Code cleanup

# Summary
Code cleanup for a few files. Quality of life update also with some tiny fixes

## What I did here
- Cleaned up TFLite_detection_stream file to include a main
- Cmd line arguments are now parsed in main, and relevant global variables are set
- Flask app is now run/started in main
- Moved VideoStream class out of TFLite_detection_stream into a separate module
- Fixed backend bug where total disk space was being used for total memory 
- **Note:** ran pip install -U numpy inside venv to fix local env

## How to test
1.  Pull branch onto local
2. source venv
3. run python command below to start app and ensure it starts normally
```
python3 TFLite_detection_stream.py --modeldir=Sample_TFLite_model --edgetpu --streamurl='URL_HERE' --threshold=0.5 --resolution='704x480' --ip=IP_HERE --port=PORT_HERE
```
4. Maybe check if FPS is worse than before
